browser/flagr-ui/src/App.vue

```diff
@@ -39,16 +39,17 @@ const version = require("../package.json").version || "1.0.0";
 export default {
   name: "app",
   data: () => ({
-    version,
-  }),
+    version
```

Comment:
I would leave the commas at the end since it avoids conflicts on future changes for lines that are unrelated...